projectfiles/VC12/README.md

```diff
@@ -82,7 +82,10 @@ folders, all version numbers are minimal requirements:
       into the 'lib' *General Folder*.
 
 
-   3. libboost-1.48.0 ( http://www.boost.org/ )
+   3. libboost-1.50.0 ( http://www.boost.org/ )
+
+      Note these instructions are still referring to Boost 1.48.0, and are so
```

Comment:
The instructions are more severely outdated because of Wesnoth's dependencies themselves having changed...

Reply:
Reject with "changes requested" then?

Reply:
Well, instructions being outdated aren't really your fault, so it's not a valid reason to reject this PR.

Reply:
I assumed, wrongly, everything was OK, and mindlessly `grep`ed and replaced without even reading the files.

As it was already pointed out, the title of the PR is misleading (actually both commits too, for the same reason). I find that valid enough for rejection. These are all what I think we could do about this, bear in mind I'm noob at code review:

* Roll it back: close, either by unacceptable (misleading wording) or if anyone else is already taking care of or, broadly speaking, somehow assigned to it. Or even because of not actually meaningful enough, e.g.: project is planning to upgrade its Boost level in a shortest enough time.

* Roll it forward: stack commits, at least another one, until PR is acceptable. But since I don't generally care about Microsoft, somebody, rather than me, would have to state a target scope that I will take, or will not. In case of not taking, I assume the right thing is to unlock the PR for you to be free to use it only in case that seems useful or appropriate to you.

* Roll it over itself: squash and rebase until have something semantic and beautiful enough to be undoubtedly acceptable for merging. But I'm noob at this GitHub peer review system. I don't know how good it deals with rebase interactive followed by forced push to feature branch. The externally defined target scope thing introduced by the previous bullet still applies this.

* Simply accept it: If someone else is already somehow assigned to Microsoft-related documentation, and s/he loves it, maybe there is no point in asking me to deal with it.

Your call.

Reply:
"Simply accept it" is what I prefer. I don't like increasing the scope of a pull request. I'm willing to merge any PR that is an improvement regardless of what related changes would be useful.

Regarding "roll it over itself", it works perfectly. I often use history rewriting and force pushes in my own pull requests. So don't worry about it breaking GitHub. (But be wary of the risk that by rewriting history you may accidentally erase changes which haven't been merged.)

Reply:
If you are using a branch on private fork you can 'roll over' the PR. For that, I use git rebase, usually with --interactive. It allows me to re-order the commits, merge commits, delete or add. When I have everything looking right I `git push -f` over my private fork branch and github figures it out,

Reply:
Well, I rewrite history even in this upstream repository. :D (Although obviously not in master.)